DESIGN.md

```diff
@@ -36,28 +36,28 @@ OSM ships out-of-the-box with all necessary components to deploy a complete serv
 
 
 ## OSM Components & Interactions
-![OSM Components & Interactions](/docs/development_guide/osm-components-and-interactions.png)
+![OSM Components & Interactions](/docs/images/osm-components-and-interactions.png)
 
 ### Containers
 When a new Pod creation is initiated, OSM's
-[MutatingWebhookConfiguration](/charts/osm/templates/mutatingwebhook.yaml)
+[MutatingWebhookConfiguration](/pkg/injector/webhook.go#L422)
```

Comment:
I don't think we should be referencing specific lines in the docs; these can change very frequently, and we may or may not remember to update them. Please remove exact line references in the code. If you wish to keep references, lets point to specific files and not lines

Reply:
Applies to everything below as well